doc/build/faq/connections.rst

```diff
@@ -16,7 +16,7 @@ How do I pool database connections?   Are my connections pooled?
 ----------------------------------------------------------------
 
 SQLAlchemy performs application-level connection pooling automatically
-in most cases.  With the exception of SQLite, a :class:`_engine.Engine` object
+in most cases.  For all included dialects, a :class:`_engine.Engine` object
```

Comment:
For all included dialects (with the exception of the SQLite dialect when using a "memory" database)